### PR TITLE
Publishing v0 of In skill purchase component

### PR DESCRIPTION
**Pull request by prachibhav**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by AndrewKeating-Amzn — COMMENTED** [review]
Don't need the .DS_Store files, can remove and add to .gitignore

File: in-skill-purchase/gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.*
```
Comment (AndrewKeating-Amzn) [inline in-skill-purchase/gitignore:1]: this file needs to be ".gitignore"

File: in-skill-purchase/Config [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package.ISPReusableComponent = {
```
Comment (AndrewKeating-Amzn) [inline in-skill-purchase/Config:1]: this is for brazil build, no need to expose publicly

File: in-skill-purchase/response/prompts/AlexaConversationsBye/document.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (AndrewKeating-Amzn) [inline in-skill-purchase/response/prompts/AlexaConversationsBye/document.json:1]: We don't need any of these default prompts in skill components, only prompts used in the ACDL will actually ever be used

**Review by Aaditya-Amzn — APPROVED** [review]
Looks good to me

**Pull request merged**